source/common/formatter/substitution_formatter.cc

```diff
@@ -7,6 +7,7 @@
 #include <cstdlib>
 #include <memory>
 #include <regex>
+#include <stdexcept>
```

Comment:
@betsybutton  I realized this is an unused include after the refactoring. Could you please put a quick fix on the same ticket for this?

Reply:
Oh, nice catch! Absolutely.

Reply:
@visualzhou Here's the PR: https://github.com/10gen/envoy-serverless/pull/17